docker-compose.yml

```diff
@@ -120,3 +134,6 @@ services:
 
 volumes:
   s3-folder: null
+
+networks:
+  importexportstompnet:
```

Comment:
[question]
Why add a custom network?

Reply:
To follow our django-template, does make sense?

Reply:
Removed!